deploy/overlays/webhook/role.yaml

```diff
@@ -1,3 +1,3 @@
 ---
-- path: /rules/2
+- path: /rules/3
```

Comment:
In case someone is going wtf is this change..because I changed the spo role by adding a new permission, this overlay would have removed a permission at the wrong index:
```
diff --git a/deploy/webhook-operator.yaml b/deploy/webhook-operator.yaml
index cad32631..ed2ae5f8 100644
--- a/deploy/webhook-operator.yaml
+++ b/deploy/webhook-operator.yaml
@@ -2171,9 +2171,9 @@ rules:
   - list
   - watch
 - apiGroups:
-  - ""
+  - admissionregistration.k8s.io
   resources:
-  - services
+  - mutatingwebhookconfigurations
   verbs:
   - create
```